packages/forklift-console-plugin/src/modules/Plans/views/list/components/PlanStatusVmCount.tsx

```diff
@@ -28,12 +33,18 @@ export const PlanStatusVmCount: React.FC<PlanStatusVmCountProps> = ({
         return <ExclamationTriangleIcon />;
       case 'danger':
         return <ExclamationCircleIcon />;
+      default:
```

Comment:
Why the `canceled` VM status is not passed to `PlanStatusVmCount` and treated as default?